src/Web/Sfa.Das.Sas.Web/Views/Apprenticeship/Search.cshtml

```diff
@@ -5,25 +5,25 @@
     ViewBag.PageID = "search-provider-home";
 }
 <main id="content" role="main">
-    <div class="grid-row">
+    <div class="grid-row apprenticeship-search">
         <div class="column-two-thirds">
-            <a class="link-back" href="/Apprenticeship/ApprenticeshipOrProvider">Back</a>
+            <a class="link-back" href="/">Back</a>
             <h1 class="heading-xlarge">
                 Find apprenticeship training
             </h1>
-            @using (Html.BeginForm("SearchResults", "Apprenticeship", FormMethod.Get, new { @class = "search-box" }))
+            @using (Html.BeginForm("SearchResults", "Apprenticeship", FormMethod.Get, new { @class = "search-box apprenticeship-search-form" }))
             {
                 <div class="form-group">
                     <label for="keywords">
-                        <span class="form-label-bold">Search by keywords</span>
-                        <span class="form-hint">Can include job title or apprenticeship</span>
+                        <span class="form-label">Apprenticeship name or job role</span>
                     </label>
                     <span class="error-message"></span>
                     <input type="search" id="keywords" name="Keywords" class="text-box form-control" maxlength="200">
                 </div>
-                <input class="button" id="submit-keywords" type="submit" value="Search" />
+                <input class="button" id="submit-keywords" type="submit" value="Search Apprenticeship Training" />
```

Comment:
Should this be 'Search apprenticeship training' ? We generally only add a capital to the first word